scipy/special/_generate_pyx.py

```diff
@@ -1153,7 +1156,7 @@ def _generate_from_outargs_and_return(self):
                 call = double_complex_from_npy_cdouble(call)
             call = "{}[0] = {}".format(self.outvars[0], call)
             body.append(sp + call)
-            body.extend(map(lambda x: sp + x, casts))
+            body.extend([sp+x for x in casts])
```

Comment:
Are spaces around the operator required, `body.extend([sp + x for x in casts])`?

Reply:
added spaces